api/src/test/java/org/pmiops/workbench/testconfig/TestCdrJpaConfig.java

```diff
@@ -1,60 +0,0 @@
-package org.pmiops.workbench.testconfig;
-
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Profile;
-import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
-import org.springframework.jdbc.datasource.DriverManagerDataSource;
-import org.springframework.orm.jpa.JpaTransactionManager;
-import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
-import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
-import org.springframework.transaction.annotation.EnableTransactionManagement;
-
-import javax.persistence.EntityManagerFactory;
-import javax.sql.DataSource;
-import java.util.Properties;
-
-@Configuration
-@EnableJpaRepositories(basePackages = { "org.pmiops.workbench.cdr" })
-@EnableTransactionManagement
-public class TestCdrJpaConfig {
```

Comment:
We don't need this anymore due to CdrDbConfig?

Reply:
This is no longer need for database test for cdr specific daos. Since we no longer need to prefix our native sql statements with 'cdr.'